binscripts/rvm-installer

```diff
@@ -219,7 +218,7 @@ __rvm_curl()
 )
 
 rvm_error()  { printf "ERROR: %b\n" "$*"; }
-__rvm_which(){   which "$@" || return $?; true; }
+__rvm_which(){ command -v "$@" 2>/dev/null; }
```

Comment:
but we want to keep the same code in the other places, so `__rvm_which` function in the other place - would need to be changed, and then we get a problem, as there we can work with `zsh`